tests/utils/test_evaluator.py

```diff
@@ -36,46 +41,52 @@ def reward_config():
 
 
 @pytest.fixture
-def satisfaction_classifier():
+def satisfaction_classifier() -> SatisfactionClassifierSVM:
     """Tests satisfaction classifier init.
 
     Also used as a fixture for the tests.
     """
     return SatisfactionClassifierSVM()
 
 
-def test_init(dialogues, reward_config):
+def test_init(
+    annotated_dialogues: List[Dialogue], reward_config: Dict[str, Any]
+) -> None:
     """Tests evaluator initialization.
 
     Args:
-        dialogues: Testing dialogues.
-        reward_config: _description_
+        annotated_dialogues: Test dialogue object.
+        reward_config: Test reward config.
     """
-    Evaluator(dialogues=dialogues, reward_config=reward_config)
+    Evaluator(dialogues=annotated_dialogues, reward_config=reward_config)
 
 
-def test_avg_turns(dialogues, reward_config):
-    """Tests avg turns method.
+def test_avg_turns(
+    annotated_dialogues: List[Dialogue], reward_config: Dict[str, Any]
+) -> None:
+    """Tests avg_turns method.
 
     Args:
-        dialogues: Test dialogue object.
+        annotated_dialogues: Test dialogue object.
```

Comment:
Nit: objects.

Reply:
It is a fixture object therefore I would leave it singular.